contracts/verifiers/Verifier.sol

```diff
@@ -601,13 +613,14 @@ abstract contract Verifier is IVerifier, ContextUpgradeable {
     {
         VerifierStorage storage s = _getVerifierStorage();
         Proof storage proof = s._proofs[requestId][sender];
+        ProofEntry storage lastProofEntry = proof.proofEntries[proof.proofEntries.length - 1];
```

Comment:
Here you have to check previously if `proof.isVerified`. Otherwise will revert with panic if we call this `getRequestProofStatus` without any entries in the proof.

Reply:
Fixed